Change log target from `log_panics` to `panic`

Currently when logging it uses the default target, which is the module name `log_panics`. But in my opinion it would be nicer to change this to `panic`, since the panic itself really has nothing to do with the module.

```
2017-08-04T12:32:58.203374+00:00 [ERROR] log_panics: thread 'tests::log_output' panicked at 'oops'
// vs
2017-08-04T12:32:58.203374+00:00 [ERROR] panic: thread 'tests::log_output' panicked at 'oops'
```

Would a change like that be acceptable?

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -64,14 +64,14 @@ pub fn init() {
 
         match info.location() {
             Some(location) => {
-                error!("thread '{}' panicked at '{}': {}:{}{:?}",
+                error!(target: "panic", "thread '{}' panicked at '{}': {}:{}{:?}",
                        thread,
                        msg,
                        location.file(),
                        location.line(),
                        Shim(backtrace));
             }
-            None => error!("thread '{}' panicked at '{}'{:?}", thread, msg, Shim(backtrace)),
+            None => error!(target: "panic", "thread '{}' panicked at '{}'{:?}", thread, msg, Shim(backtrace)),
         }
     }));
 }

